styles/global.scss

```diff
@@ -1,3 +1,5 @@
+@use "./partials/mixin.scss" as *;
+@use "./partials/variables.scss" as *;
 @import 'https://fonts.googleapis.com/css2?family=Titillium+Web:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700&display=swap';
```

Comment:
It looks great man, 
but about the font, i think you can apply the font to related HTML tags here
**REASON** 
it makes font easy to use everywhere in the app and consistent